geo.json

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
Remember to remove this file in the final version of this PR (since all will go to `out.json`).